src/ambianic/pipeline/store.py

```diff
@@ -171,18 +175,52 @@ def process_sample(self, **sample) -> Iterable[dict]:
                 yield processed_sample
 
     def notify(self, save_json: dict):
-        if self.notification is None:
-            return
-        log.debug("Sending notification(s)..")
         # TODO extract inference data
+        if save_json['inference_result'] is None:
+            return
+
         for inference_result in save_json['inference_result']:
             data = {
                 'id': save_json['id'],
                 'label': inference_result['label'],
                 'confidence': inference_result['confidence'],
                 'datetime': save_json['datetime'],
             }
-            notification = Notification(data=data, providers=self.notification_config["providers"])
+
+            try:
+                premiumFile = pkg_resources.resource_filename(
+                    "ambianic.webapp", "premium.yaml")
+                with open(premiumFile, "r") as file:
+                    configFile = yaml.safe_load(file)
+                    userId = configFile['credentials']["USER_AUTH0_ID"]
+                    endpoint = configFile['credentials']["NOTIFICATION_ENDPOINT"]
+                    if (userId):
+                        post(
+                            url='{0}/notification'.format(endpoint),
+                            json={
+                                'userId': userId,
+                                'notification': {
+                                    'title': 'Ambianic.ai New {0} event'.format(
+                                        data['label']),
+                                    'message': 'New {0} detected with a {1} confidence level'.format(
+                                        data['label'],
+                                        data['confidence']),
+                                }})
+
+            except FileNotFoundError:
+                log.info("Error locating file")
+
+            except TypeError:
+                log.info("Auth0 key not found")
```

Comment:
@vickywane errors are usually logged with at least warn or error level, not info.

Reply:
Noted. I would correct that. 

However, in development, I can only see items logged using `log.info()`